### PR TITLE
Merge 4.2 into master

### LINKED ISSUE #630: Release 4.2.2 Ansible
- Release 4.2.2 Ansible 
  - [x] Publish draft release 
  - [x] Merge 4.2 into stable
  - [ ] Merge 4.2 into master

### PR DESCRIPTION
**Pull request by lastranikos**
This PR closes #630

### CODE REVIEW

**Review by teddytpc1 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ These playbooks install and configure Wazuh agent, manager and Elastic Stack.
 | Wazuh version | Elastic | ODFE   |
 |---------------|---------|--------|
 | v4.3.0        | 7.10.2  | 1.13.2 |
+| v4.2.1        | 7.10.2  | 1.13.2 |
 | v4.2.0        | 7.10.2  | 1.13.2 |
```
Comment (rauldpm) [inline README.md:20]: There is no v4.2.2?

**Review by rauldpm — CHANGES_REQUESTED** [review]
Minor changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ All notable changes to this project will be documented in this file.
 
 - Update to [Wazuh v4.3.0](https://github.com/wazuh/wazuh/blob/v4.3.0/CHANGELOG.md#v430)
 
+## [v4.2.1]
+
+### Added
+
+- Update to [Wazuh v4.2.1](https://github.com/wazuh/wazuh/blob/v4.2.1/CHANGELOG.md#v420)
+
 ## [v4.2.0]
```
Comment (teddytpc1) [inline CHANGELOG.md:16]: Same comment as Raul's in the README.md. 4.2.2 should be listed in this file.

**Review by teddytpc1 — CHANGES_REQUESTED** [review]

**Comment by lastranikos** [comment]
This PR is fixed at https://github.com/wazuh/wazuh-ansible/pull/635

**Pull request closed without merging**